src/AutoRest.CSharp/LowLevel/Extensions/Snippets.ExampleValues.cs

```diff
@@ -117,27 +118,33 @@ private static CodeWriter AppendFrameworkTypeValue(this CodeWriter writer, Input
                     switch (serializationFormat)
                     {
                         case SerializationFormat.DateTime_Unix:
-                            if (rawValue.RawValue is int or long)
-                                return writer.Append($"{typeof(DateTimeOffset)}.FromUnixTimeSeconds({rawValue.RawValue:L})");
+                            long? time = rawValue.RawValue switch
+                            {
+                                int i => i,
+                                long l => l,
+                                _ => null
+                            };
+                            if (time != null)
+                                return new InvokeStaticMethodExpression(typeof(DateTimeOffset), nameof(DateTimeOffset.FromUnixTimeSeconds), new[] { Long(time.Value) });
```

Comment:
`DateTimeOffsetExpression.FromUnixTimeSeconds`

Reply:
Thank you. I added a method for `DateTimeOffset.Parse` as well.